public/locales/es/translation.json

```diff
@@ -0,0 +1,193 @@
+      "parent_element": "Elemento padre",
+      "remove": "Eliminar",
+      "remove_flavour": "Sabor a eliminar",
+      "remove_flavour_button": "Eliminar sabor y todos los descendientes"
+  },
+  "reset": {
+      "content": "Reiniciar tu Smorgasbord cargará el lienzo predeterminado: tus cambios se perderán. ¿OK?",
+      "confirm": "Sí, reiniciar",
+      "cancel": "No, cancelar"
+  },
+  "footer": {
+      "disclaimer": "<a href=\"https://github.com/duizendnegen/sunburst-smorgasbord\">Open Source</a>; hecho con <a href=\"https://d3js.org/\">D3</a>, basado en su <a href=\"https://observablehq.com/@d3/sunburst\">diagrama Sunburst</a>.<br />Sin cookies, sin seguimiento, sin datos recopilados. Se utiliza almacenamiento local para asegurarse de que no tenga que comenzar de nuevo cada vez que cargue el sitio web. Genial.",
+      "languages": "Disponible en",
+      "languages_german": "Alemán",
+      "languages_english": "Inglés",
+      "languages_spanish": "Español"
+  },
+  "faq": {
+      "whats_this": "¿Qué es esto?",
+      "whats_this_content": "Esta es una herramienta para ayudarte a discutir y explorar las relaciones que te gustaría tener con tus vínculos. Está inspirada en el <a href=\"//www.multiamory.com/podcast/339-the-smorgasbord-of-relationships\">Smorgasbord de Anarquía Relacionales</a>, que fue desarrollado inicialmente por Lyrica Lawrence y Heather Orr en 2016.",
+      "how_to_use": "¿Cómo se usa?",
+      "how_to_use_content_1": "Concéntrate en una relación específica que tengas en tu vida. Luego, echa un vistazo a las palabras escritas en cada panel, y piensa si son relevantes para tu relación, o si te gustaría que lo fueran. Revela los colores de tu relación haciendo clic en los paneles que te gusten.",
+      "how_to_use_content_2": "Exactamente cómo uses esta herramienta depende de ti, pero funciona mejor cuando compartes los resultados con tus vínculos. Puedes usarlo junto con tus vínculos o cada cual usarlo de forma independiente y después compartir los resultados. El punto es tener una conversación.",
+      "whats_ra": "¿Qué es la Anarquía Relacional?",
+      "whats_ra_content": "La Anarquía Relacional es una forma de abordar las relaciones de manera intencional. En lugar de asumir cómo debería ser una relación basándose en su etiqueta (por ejemplo, cónyuge, amante, padre, amistad), defines activamente cómo quieres que sea la relación. Descrita por primera vez por Andie Nordgren en 2006, <a href=\"https://log.andie.se/post/26652940513/the-short-instructional-manifesto-for-relationship\">El Manifiesto Instructivo Corto para la Anarquía Relacional</a> es un buen lugar para comenzar."
+  },
+  "flavours": {
+      "caregiving": "Cuidado",
+      "collaboration": "Colaboración",
+      "communication": "Comunicación",
+      "companionship": "Compañerismo",
+      "creativity": "Creatividad",
+      "domesticity": "Vida doméstica",
+      "each_other": "Le un a le otre",
```

Comment:
I don't understand this translation

Reply:
"El uno al otro" in male language. In neutral it would be "Le une a le otre". I missed the e in une. Sorry